build.sh

```diff
@@ -13,6 +13,8 @@ export FC=${FC:-mpif90}
 
 export ESMFMKFILE=${ESMFMKFILE:?"Please set ESMFMKFILE environment variable"}
 
+CMAKE_FLAGS="-DAPP=ATM"
+
```

Comment:
This should be reverted before we start the commit process

Reply:
Reverted.